### DISCUSSION TITLE
How do i append to the quickfix list ?

### DISCUSSION THREAD

**Discussion by ArbitRandomUser** [ibhagwan/fzf-lua#1656] [General]
basically the title , 
i want to append to append an entry to the quickfix list ,
currently alt-q will replace the previos entry

**Comment by ibhagwan** [comment]
> basically the title , i want to append to append an entry to the quickfix list , currently alt-q will replace the previos entry

No such functionality exists, you can mimic that by creating a custom function that reads rhe current list, adds what you want and calls `setqflist` with the new list.

Have you tried `:FzfLua quickfix_stack`?

**Comment by ArbitRandomUser** [comment]
thank you ,
